windows/powershell_dsc.yml

```diff
@@ -4,6 +4,18 @@
   gather_facts: false
 
   tasks:
+    - name: Setup PsGallery
+      ansible.windows.win_powershell:
+        script: |
+          $nuget_version = (Get-PackageProvider -Name NuGet -ListAvailable).version
+          $nuget_target_version = '2.8.5.201'
```

Comment:
for proper version comparison, should this be something like `$nuget_target_version = [Version]::new('2.8.5.201')` instead?  i ask as someone with no powershell proficiency, but i've seen version comparisons get strange with standard greater/less than operators in bash before so i thought i'd look for a [powershell example](https://stackoverflow.com/questions/56442941/compare-the-software-version-number-in-powershell).  i'm not sure if specifying the variable as a `Version` type would cause issues with the `Install-PackageProvider` call below this.

Reply:
PowerShell is typically smart enough to do the conversion on the fly by I don't have an issue hard typing it.

Reply:
addressed in latest push